.github/actions/sanitize/action.yml

```diff
@@ -49,4 +49,4 @@ runs:
       shell: bash
       run: |
         cd ${{ inputs.working-directory }}
-        RUST_BACKTRACE=1 cargo nextest run --profile ci
+        CARGO_BUILD_JOBS=2 RUST_BACKTRACE=1 cargo nextest run --profile ci
```

Comment:
This solves the problem of builds running out of memory on linux-arm, at the cost of reduced parallelism. I see 8vcpu in the names. Let's try 4? or different approach?

Reply:
Our runners have 12GB of RAM - kinda worrying if we're managing to exhaust that

Reply:
Yes I think it's probably one very specific thing (test, dependency, ?) that has pathological behaviour, I've worked on projects with more tests that didn't have that problem.

Reply:
I've seen `cargo test` in CLI take a very long time locally as well, for anecdotal evidence.